wordcloud_style.py

```diff
@@ -17,10 +17,24 @@ def select_colour(self, word, value, word_to_values):
 class QuantizedColourSelector(ColourSelector):
     """Apply colour based on a user provided list."""
 
+    def __init__(self, colour_list, max_value):
+        self.colour_list = colour_list
+        self.max_value = max_value
```

Comment:
Creating the max_value attribute here is correct. The starting value doesn't matter because it'll be replaced by on_begin_draw(), so I'd just use a starting value of 0.